DigitalLibraryofTennessee/DLTN_XSLT
# Add sample record & transform changes for Rhodes Oral Histories.

**GitHub Issue**: [DLTN_XSLT-111](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/111)

* [Metadata mapping](https://dltn-technical-docs.readthedocs.io/en/latest/mappings/crossroads.html#col-10267-30936-crossroads-to-freedom-oral-history-collection)

## What does this Pull Request do?

This pull requests adds a sample record for Rhodes Oral Histories and makes modifications to the default template that work for both Friends and Family and Oral Histories.

## What's new?

* A sample record to test against
* Adds a namespace for local things to keep things simple
* Adds a list of acceptable params for typeOfResource and a matching incoming value
* Changes the thumbnail xpath selector so that only one thumbnail is ever selected
* Creates a default rule for mapping descriptions to abstract.  This will need to be enhanced later.
* Maps a link to the streaming video when something is hosted on vimeo
* Ensures Rhodes College is never listed as publisher
* Changes typeOfResource rule so that a value must match an existing param or nothing happens

## How should this be tested?

1. Run updated stylesheet versus the incoming sample record.
2. Check for validity.
3. Check that mapping table is accurate.


## Additional Notes:

I need to update the mapping table so that the streaming portion is correct.

## Interested parties

@mlhale7
